### ISSUE TITLE
Add support for Android TV App

### ISSUE THREAD

**Issue by Mis1ify** [jellyfin/jellyfin-plugin-intros#10]
Plugin works perfectly, however it does not work in the Android TV Jellyfin app.

**Comment by BrianCArnold** [comment]
Make sure that Cinema Mode is turned on in the app. I think it's turned off by default. I don't have an Android TV device to verify this on.

**Comment by Mis1ify** [comment]
I have cinema mode turned on, but thanks anyway.

**Comment by BrianCArnold** [comment]
I found an old SBC that I can install the app on, I'll try to see if I can replicate the issue.

**Comment by TomGaming109** [comment]
Hey, any look with this? I just get player error occured, retrying..

Would love to have it work on the TV!

**Comment by BrianCArnold** [comment]
I've tested this on on the current `master` branch of the Android TV app, and confirmed that it does not work on TV episodes, however it does work on Movies.

The code at https://github.com/jellyfin/jellyfin-androidtv/blob/6af461e2fbb445a0de9096bf4ceeb5ba9f4178e2/app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java#L1588 looks to be specifically written to only playback intros on Movies, as the the `allowIntros` argument is only true when the base item type is "Movies".

@FaboThePlayer, can you confirm if your environment is working with Movies? If it is, this should be an issue filed jellyfin/jellyfin-androidtv. 

@TomGaming109 jellyfin/jellyfin-androidtv#2163 looks related. Can you check if you're on v0.14.5 or later? It was first included on v0.15.0-beta.1, and backported to v0.14.5.

**Comment by BrianCArnold** [comment]
@TomGaming109 Another possible cause occurs to me, are you using .mp4s or .mkvs with h264, or some other container and codec? I did run into this issue when I tried playing some intros on my android TV emulator in Android Studio, but I didn't see any issues when running it on my physical TV.

**Comment by Mis1ify** [comment]
> @FaboThePlayer, can you confirm if your environment is working with Movies? If it is, this should be an issue filed jellyfin/jellyfin-androidtv.

Yup, it's working with movies. Is there any workaround for that until they update the app?
